firedrake/slate/slac/compiler.py

```diff
@@ -559,7 +582,8 @@ def metaphrase_slate_to_cpp(expr, temps, prec=None):
         return parenthesize(result, expr.prec, prec)
 
     elif isinstance(expr, Action):
-        tensor, c = expr.operands
+        tensor, = expr.operands
+        c = expr.actee
         # Extra check
         assert isinstance(c, Coefficient)
```

Comment:
You already assert that `actee` is a `Function` in the `Action` contructor.  Probably no need to do so again here.